user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/GQIEditableRow.md

```diff
@@ -96,6 +96,8 @@ The value, of type `object`, of the cell matching the provided column.
 
 Gets the value for the cell of the specified column.
 
+Available from DataMiner 10.3.4/10.4.0 onwards.
```

Comment:
Was this added with RN 35734? If yes, I'd also add this RN number in comment, as this can help us to quickly pinpoint what introduced a specific change. (This would also apply for the other instances below.)
```suggestion
Available from DataMiner 10.3.4/10.4.0 onwards.<!-- RN 35734 -->
```